cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -387,7 +387,7 @@ macro(configure_sdk_unix name architectures)
           message(FATAL_ERROR "unsupported arch for OpenBSD: ${arch}")
         endif()
 
-        string(REPLACE "[-].*" "" openbsd_system_version ${CMAKE_SYSTEM_VERSION})
+        string(REGEX REPLACE "[-].*" "" openbsd_system_version ${CMAKE_SYSTEM_VERSION})
```

Comment:
@3405691582, is this actually needed for OpenBSD?  AFAICT, OpenBSD does not use these version suffixes in the output of `uname -r`.

See [here](https://github.com/openbsd/src/blob/69d7fcb105738c33ad53326628c1c1d8d567b36a/sys/conf/newvers.sh#L82), where `osrelease` is populated with just `${osr}`.

Reply:
I think I've confirmed this is not needed on OpenBSD:

```
# uname -a
OpenBSD fugu.my.domain 7.2 GENERIC#728 amd64
# uname -r 
7.2
```

And cmake 3.24.2 reports "7.2" for `${CMAKE_SYSTEM_VERSION}`.

So I'll go ahead and remove the regex replace for OpenBSD.

Reply:
Sorry for holding you up here.

Reply:
No worries, thanks for your help!